### DISCUSSION TITLE
Does `rustic check` support `--read-data-subset` ?

### DISCUSSION THREAD

**Discussion by kapitainsky** [rustic-rs/rustic#1001] [Q&A] [answered]
Does `rustic` atm support `--read-data` only? No options to check parts of the repo only (`--read-data-subset`)?

It is only for clarification as no big deal if not implemented yet - one can always fall back to `restic` for this particular task.

**Comment by aawsome** [comment, marked as answer]
see #251 
It is a big deal for cold storages.

However, when using a cloud storage the data integrity is typically job of the provider - and you can still do some manual check by checking the SHA256 if any repository file against its filename.

**Reply by kapitainsky** [reply]
Thx for answering. 
Yes cold storage is special case indeed.

It is clear for me now where we are:)
